### PR TITLE
TRUNK-5942 Adding curly braces to Control structures

### PR DESCRIPTION
**Pull request by miirochristopher**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5942

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.
- [ ] I have **added tests** to cover my changes. 
- [ ] I ran `mvn clean package` right before creating this pull request and added all formatting changes to my commit.
- [ ] All new and existing **tests passed**.
- [ ] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: api/src/main/java/org/openmrs/util/HandlerUtil.java [hunk 1]
```diff
@@ -78,8 +78,11 @@ public boolean equals(Object obj) {
 			}
 			if (type == null) {
 				return other.type == null;
-			} else
+			}
```
Comment (sherrif10) [inline api/src/main/java/org/openmrs/util/HandlerUtil.java:81]: Thanks @miirochristopher for working on this, could you remove some spaces, Also try to  do identation

File: api/src/main/java/org/openmrs/module/ModuleClassLoader.java [hunk 2]
```diff
@@ -530,8 +530,9 @@ protected void modulesSetChanged() {
 	 */
 	public void dispose() {
 		if (log.isDebugEnabled())
+		{
```
Comment (sacull) [inline api/src/main/java/org/openmrs/module/ModuleClassLoader.java:533]: For readability in Java, the brace "{" is on the same line as the previous line of code. If some people write on the same line, the other part in the new line, the code will become less readable.

File: api/src/main/java/org/openmrs/module/ModuleClassLoader.java [hunk 3]
```diff
@@ -909,8 +910,9 @@ protected URL findResource(final String name, final ModuleClassLoader requestor,
 		if (requiredModules != null) {
 			for (Module publicImport : requiredModules) {
 				if (seenModules.contains(publicImport.getModuleId()))
+				{
```
Comment (sacull) [inline api/src/main/java/org/openmrs/module/ModuleClassLoader.java:913]: Same as above.

File: api/src/main/java/org/openmrs/util/HandlerUtil.java [hunk 4]
```diff
@@ -78,8 +78,11 @@ public boolean equals(Object obj) {
...
+			else
+			{
```
Comment (sacull) [inline api/src/main/java/org/openmrs/util/HandlerUtil.java:83]: Again same as above.

**Review by sacull — CHANGES_REQUESTED** [review]

**Comment by miirochristopher** [comment]
Thanks for the feedback @sherrif10 and @sacull. I made commits to correct the indentation and remove spaces.

**Comment by sacull** [comment]
Great job, thanks @miirochristopher. When the PR is ready, it is worth informing about it in JIRA. To do this, use the "Request Code Review" button and add a link to this PR in the comment.

**Comment by HerbertYiga** [comment]
@miirochristopher   you can go on and squash your commits

**Review by sacull — APPROVED** [review]

**Comment by sacull** [comment]
@miirochristopher If you need help with splash commits mentioned by @HerbertYiga: https://wiki.openmrs.org/display/docs/Git-Squashing+commits

**Comment by dkayiwa** [comment]
@HerbertYiga have you ever seen this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

**Comment by miirochristopher** [comment]
@sacull and @HerbertYiga , squashing done, thanks!

**Comment by sherrif10** [comment]
@miirochristopher  did you see travis ci ?

**Comment by miirochristopher** [comment]
@sherrif10 , No I did not.

**Comment by dkayiwa** [comment]
The travis build failure is caused by something else. Am looking into it.

**Comment by sherrif10** [comment]
@dkayiwa  great thanks

**Comment by mozzy11** [comment]
@sherrif10 , @dkayiwa .
  i rebuilt the travis Build and it passed.
 had't looked at the actual cause of the innitial build failure

**Review by sherrif10 — APPROVED** [review]

**Comment by dkayiwa** [comment]
@sacull do you approve this? 😊

**Review by sacull — APPROVED** [review]

**Comment by sacull** [comment]
@dkayiwa Well, now my ego has grown since you ask me for my opinion. ;)

**Comment by sacull** [comment]
@miirochristopher For the next PR, please remember about the checklist you have in the PR description.

**Comment by dkayiwa** [comment]
You deserve it man!!! 😊 👍

**Comment by mozzy11** [comment]
hahah 😊  👍

**Comment by miirochristopher** [comment]
Thanks @sacull  @dkayiwa  @mozzy11 @HerbertYiga @sherrif10 ....That got me going. Gotta try to keep out the smoke on the next PR,  :+1: :+1: :)

**Pull request merged**